App/src/main/java/vut/fit/gja2023/app/service/SystemManagerService.java

```diff
@@ -0,0 +1,97 @@
+package vut.fit.gja2023.app.service;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import vut.fit.gja2023.app.dto.*;
+import vut.fit.gja2023.app.enums.DirectoryModeEnum;
+
+@Service
+@RequiredArgsConstructor
+public class SystemManagerService {
+
+    @Value("${systemManager.url.user}")
+    private  String SYSTEM_MANAGER_USER_URL;
+    @Value("${systemManager.url.group}")
+    private  String SYSTEM_MANAGER_GROUP_URL;
+    @Value("${systemManager.url.firewall}")
+    private  String SYSTEM_MANAGER_FIREWALL_URL;
+    @Value("${systemManager.url.directory}")
+    private  String SYSTEM_MANAGER_DIRECTORY_URL;
```

Comment:
neni tam extra medzera za private? 😊, plus ono to niesu uplne konštanty, tak by som nedaval screaming snake case(velke pismena a podtrzitko namiesto medzier), ale normalne camelCase